src/sparsezoo/v2/test.md

```diff
@@ -0,0 +1,111 @@
+# API User pathway
+
+## Download one file
+
+### Through the string arguments
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+stub = stub + "?" + "recipe=transfer_learn"
+# will read all the files from the server, except all the recipe files. The only recipe 
+# file that will be "seen" by the `model` is "transfer learn"
+model = Model(path=stub)
+model.download() # optional: may be called to download all the files, but is not required (see below). Will be omitted in further examples
+recipes: str = model.recipes.downloaded_path() # will automatically download the file (if not already downloaded) and return the path
+...
+```
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+stub = stub + "?" + "checkpoint=preqat"
+# will read all the files from the server, except for the full training folder. Training folder
+# will be identical to prequat checkpoint directory
+model = Model(path=stub)
+training: str = model.training.downloaded_path() 
```

Comment:
let's do model.training.default() to match recipes and to have a clear mapping back to folder structure of what happened here